### ISSUE TITLE
Ave pooling kernel size 4?

### ISSUE THREAD

**Issue by Zehaos** [pby5/MobileNet_Caffe#3]
Hi, Gavin
In your implement [MN_train_128_1_train.prototxt#L1484](https://github.com/pby5/MobileNet_Caffe/blob/master/MN_train_128_1_train.prototxt#L1484), ave pooling kernel size is 4.
According to the paper, ave pooling kernel size should be 7. Is it a mistake or what? Thanks.

**Comment by pby5** [comment]
because the input is 128x128, when input is 224x224, the ave pooling kernel size is 7.

**Comment by BobLiu20** [comment]
Why don't use global_pooling = true directly ? what's different from them?

**Comment by pby5** [comment]
no, they are same

**Comment by pby5** [comment]
no, they are same, when I implement it, I forget to use use global_pooling = true
